lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
It's an issue because `rest_base` is never exposed to the client. Nowhere in the JSON response for the taxonomy object does it include `rest_base`. It does, however, include the `slug` -- and `slug` is precisely what should be used to map to URIs. That's _what slug is for_. See #1381.